src/bloom.h

```diff
@@ -7,12 +7,12 @@
 #define BITCOIN_BLOOM_H
 
 #include "consensus/consensus.h"
+#include "primitives/transaction.h"
```

Comment:
Just noticed this. Can't this be replaced with a `CTransactionRef` forward declaration to avoid the header file import? Just a minor nit, of course.

Reply:
ok forward declared CTransactionRef